src/deprecated.jl

```diff
@@ -18,5 +18,5 @@ import Base: keys, values, insert!
 
 @deprecate sub(dt::AbstractDataTable, rows) view(dt, rows)
 
-@deprecate stackdf stackdt
-@deprecate meltdf meltdt
+@deprecate stackdf stack
```

Comment:
This could be considered an unrelated change but given the issues with the stackdt/meltdt maybe we should be steering people towards the non-view versions? I'm happy to revert this if others would like it reverted